cafe.rb

```diff
@@ -20,7 +20,7 @@ def take_order(menus)
   end
   print '>'
   order_number = gets.to_i
```

Comment:
各所で `-1` するのもありですが

```suggestion
  order_number = gets.to_i - 1
```

order_number を -1 してしまうのもありです。ご参考までに。